tests/test_thunderbird_urls.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+import pytest
+import requests
+
+
+@pytest.mark.skip_selenium
+@pytest.mark.nondestructive
+class TestThunerbirdURLs(object):
+
+    paths = [
+        '/thunderbird/all/',
+        '/thunderbird/releases/',
+    ]
+
+    def test_urls_ok(self, mozwebqa):
+        for path in self.paths:
+            url = mozwebqa.base_url + '/en-US' + path
+            response = requests.get(url, timeout=3)
+            assert len(response.history) <= 2
```

Comment:
We try to use the `Assert` object from `unittestzero` in our tests for assertions, rather than the native `assert`, so I suggest you do the same here for consistency.

Also, what is this test for `len(response.history) <= 2`? What exactly are we testing and why do we have that expectation?